CHANGELOG.md

```diff
@@ -11,6 +12,16 @@ All notable changes to this project will be documented in this file.
 - `0.15` Releases - [0.15](#15)
 
 ---
+## [0.20](https://github.com/wordpress-mobile/MediaPicker-iOS/releases/tag/0.21)
+Released on 2017-09-06. All issues associated with this milestone can be found using this
+[filter](https://github.com/wordpress-mobile/MediaPicker-iOS/pulls?utf8=✓&q=is%3Apr%20is%3Aclosed%20milestone%3A0.21).
+
+### Fixed
+- Some crashes and bugs: #219 #221 .
+- Fixed bugs relate to selection of assets and refresh:#225 #223 .
```

Comment:
Typo and spacing: 

`- Fixed bugs relate to selection of assets and refresh. #225 #223`

In fact, can we update the formatting of these lines to match the ones from the previous version? So:

`- Summary: #123 .` => `- Summary. #123`